frontend/lib/uast/post-parse-checks.cpp

```diff
@@ -995,6 +1098,53 @@ void Visitor::visit(const Import* node) {
   }
 }
 
+template <typename F, typename NodeType>
+static bool walkAndCheckParentStack(const std::vector<const AstNode*>& stack,
+                                    F toCall,
```

Comment:
Can you rename this formal? We already have a `toCall` that converts something to a `uast::Call*` and that confused me for a minute.